playbook.md

```diff
@@ -106,29 +106,56 @@ If you are making a substantial change (perhaps following a team conversation) p
+
+* `feature/{1234-title}` - Feature branch with story ID and a short title
+* `hotfix/{5678-title}` - Hotfix branch with story/ticket ID and short title
+
+At this point if starting a new story on a sprint, remember to hit “start” on the story in Pivotal Tracker.
+
+#### Committing
+
+We follow the [Angular Git commit message conventions](https://docs.google.com/document/d/1QrDFcIiPjSLDn3EL15IJygNPiHORgU1_OOAqWjiDU5Y/edit#). At the very least, commits should have a header including a type and subject using imperative, present tense (i.e. change, not changed). 
+
+You should aim to commit little and often, making sure you commit small bits of working functionality each time. Commits with only a subject are fine, however using the commit message body is sometimes useful for explaining the reasoning behind a piece of code or a particular approach to solving a problem.
+
+#### Code style 
+
+We write code in a consistent way to ensure code is well-structured and easy to follow for the rest of the team. Similar to the guidelines laid out in [Thoughtbot’s style guide](https://playbook.thoughtbot.com/#style-guide) approach this guidance as a way to code on present and future projects, not something to retroactively add to existing projects. When working on existing projects or joining a project, make sure to follow the code style that is already in place.
+
+While it’s useful to write your own style guides, it’s often easier to follow some guidelines that already exist. To that effect our code should be written to the following guidelines:
+
+* PHP - [PSR-2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)
+* Sass - [Sass Guidelines - Syntax and formatting](http://sass-guidelin.es/#syntax--formatting)
+
+#### Pull requests and code reviews
+
+When you have finished a piece of work on a branch you should make a pull request and ask for your code to be reviewed. Code reviews are important to help maintain consistency in the code we write, ensure we push well-reasoned and bug-free code to production, and help each other learn throughout a project.
+
+Pull requests should be made when a feature is completed by a developer at the same time that it is passed to a delivery manager to review. Once the reviewing developer, delivery manager, and client are happy with the feature, it can then be merged with the `master` branch and deployed to production.
+
+When performing a code review, you should look to make sure:
+* The chosen solution is appropriate
+* Code is written in a way that is easy to understand or explained via relevant comments
+* Code follows our [code style guides](#code-style)
+
+#### Deploying
+
+We maintain two environments for deployments, `testing` and `production`. Deployments to `testing` should be done throughout the sprint ready for sign off from the client. Deployments to `production` should be made once code has been signed off from the client and passed a code review from another developer.
```

Comment:
Yes, but that's everything except TVH. I don't want to introduce too much ambiguity. Might need a section at the top for "big projects set their own rules"?